### PR TITLE
Access level

### PR DESCRIPTION
**Pull request by nicoolle**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: EPlast/EPlast.BLL/DTO/ActiveMembership/AccessLevelTypeDTO.cs [hunk 1]
```diff
@@ -13,8 +13,41 @@ public enum AccessLevelTypeDTO
...
+        [Description("Доступ члена проводу організації, Діловод Станиці")]
+        LeadershipMemberForCitySecretary,
+
+        [Description("Доступ члена проводу організації, Голова Округи")]
+        LeadershipMemberForOkrugaHead,
+
+        [Description("Доступ члена проводу організації, Заступник Голови Округи")]
+        LeadershipMemberForOkrugaHeadDeputy,
+
+        [Description("Доступ члена проводу організації, Діловод Округи")]
```
Comment (sharix4) [inline EPlast/EPlast.BLL/DTO/ActiveMembership/AccessLevelTypeDTO.cs:49]: this user does not have access  [Description("Доступ члена проводу організації, Діловод Округи")] It should be   Діловод Округи

**Review by sharix4 — CHANGES_REQUESTED** [review]

File: EPlast/EPlast.BLL/DTO/ActiveMembership/AccessLevelTypeDTO.cs (hunk 1 above)
Comment (nicoolle) [reply EPlast/EPlast.BLL/DTO/ActiveMembership/AccessLevelTypeDTO.cs:49]: Just for Діловод Округи or and for Діловод Станиці, Діловод Куреня?

File: EPlast/EPlast.BLL/Services/ActiveMembership/AccessLevelService.cs [hunk 2]
```diff
@@ -67,9 +111,64 @@ public async Task<IEnumerable<string>> GetUserAccessLevelsAsync(string userId)
                 accessLevels.Add(AccessLevelTypeDTO.PlastMember.GetDescription());
             }
 
-            if (userRoles.Intersect(leadershipLevelRoles).Any())
+            if (userRoles.Intersect(leadershipLevelRoleForGoverningBodyHead).Any())
```
Comment (sharix4) [inline EPlast/EPlast.BLL/Services/ActiveMembership/AccessLevelService.cs:114]: here it is probably necessary switch case

File: EPlast/EPlast.BLL/Services/ActiveMembership/AccessLevelService.cs [hunk 3]
```diff
@@ -23,22 +23,66 @@ public AccessLevelService(IPlastDegreeService plastDegreeService, IUserManagerSe
...
-                Roles.OkrugaHeadDeputy,
-                Roles.OkrugaSecretary,
-                Roles.GoverningBodyHead,
-                Roles.GoverningBodySecretary,
+            var leadershipLevelRoleForGoverningBodyHead = new List<string>
+            {
+                Roles.GoverningBodyHead
+            };
+
+            var leadershipLevelRoleForSectorHead = new List<string>
```
Comment (sharix4) [inline EPlast/EPlast.BLL/Services/ActiveMembership/AccessLevelService.cs:31]: why do you have so many lists with 1 value

**Review by sharix4 — CHANGES_REQUESTED** [review]

**Comment by christykobyletska** [comment]
already done

**Pull request closed without merging**